src/bendpy/src/lib.rs

```diff
@@ -57,7 +57,7 @@ fn databend(_py: Python, m: &PyModule) -> PyResult<()> {
         GlobalServices::init(conf.clone()).await.unwrap();
 
         // init oss license manager
-        OssLicenseManager::init().unwrap();
+        OssLicenseManager::init(conf.query.tenant_id.clone()).unwrap();
```

Comment:
This line code format looks not pretty.

Reply:
> This line code format looks not pretty.

Are you talking about refactor code here into:

```rust
let tenant_id = conf.query.tenant_id.clone();

OssLicenseManager::init(tenant_id).unwrap();
```